src/lib/constants.py

```
@@ -97,12 +98,12 @@
     #"gpt-5.4-nano",
     "gpt-5.3-chat-latest",
     #"gpt-5.2",
-    #"gpt-5.2-chat-latest",
+    "gpt-5.2-chat-latest",
     #"gpt-5.1-chat-latest",
```

Flake8 (7.3.0)

[error] block comment should start with '# '

(E265)